pkg/addonmanager/controllers/agentdeploy/utils.go

```diff
@@ -90,6 +92,9 @@ func newManifestWork(addonNamespace, addonName, clusterName string, manifests []
 			Labels: map[string]string{
 				addonapiv1alpha1.AddonLabelKey: addonName,
 			},
+			Annotations: map[string]string{
+				common.CloudEventsDataTypeAnnotationKey: payload.ManifestBundleEventDataType.String(),
```

Comment:
We need to add this annotation all the time? I do not think we should add any additionaly annotation to existing manifestwork.

Reply:
yes, we do need this all the time. /cc @skeeey

Reply:
we need this annotation to indicate the manifest data type, but I think we may be able to enhance the sdk-go, if we don't find this annotation, we use `ManifestBundle` as a  default value?

Reply:
and for ocm/acm cases, it should be always `ManifestBundle`

Reply:
yes, we can set default value(bundle type) in sdk-go so that we don't need set it here explicitly.

Reply:
or we can give a fixed value(bundle type) now

Reply:
updated sdk-go dependency, removed this annotation.